File: 40-cost-optimization/npm-package/optimization.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function getRecommendations() {
  const res = await axios.get(`${API}/optimization/recommendations`,
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('💡 Recommendations:', res.data.data.length);
  return res.data.data;
}

File: 40-cost-optimization/python-sdk/optimization.py
import os, requests
API = "https://api.costkatana.com/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def get_recommendations():
    res = requests.get(f"{API}/optimization/recommendations",
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"💡 Recommendations: {len(res.json()['data'])}")
    return res.json()["data"]
